src/qr-code.js

```diff
@@ -57,9 +57,9 @@ export default class QrReader {
     this.onError = options.errorCallback;
 
     // Initialize attributes
-    this._video = this._createVideoElement(videoSelector)
-    this._width = this._video.width;
-    this._height = this._video.height;
+    this._video = this._createVideoElement(videoSelector);
+    this._width =  this._video.offsetWidth;
```

Comment:
offsetWidth is only working on chrome, can you change this and use clientWidth instead? same for height